docs/src/troubleshooting.md

```diff
@@ -569,3 +569,29 @@ VOLNAME=$(kubectl get pv -o json | \
 
 kubectl delete pod/$PODNAME pvc/$PODNAME pvc/$PODNAME-wal pv/$VOLNAME
 ```
+
+### Error while bootstrapping data directory
+
+If you encounter the issue that your database cannot be initialized because it crashes during the bootstraping process, it is likely that your hugepage settings are set wrong.
+
+To check whether hugepages are enabled, run `cat /proc/meminfo` and check if huge pages are configured, their size and how many are free. If they are not enabled, you can adjust your cluster configuration as follows:
+```
+  postgresql:
+    parameters:
+      huge_pages: off
+```
+If they are enabled, you need to define how much memory should be allocated on them.
+Example:
```

Comment:
Show sample meminfo and how to derive available huge pages limits from host meminfo. Warn that the host meminfo huge pages limit is a shared pool across all containers, so if not using host/pod (anti)affinity to prevent multiple postgres pods on a kube node, must factor in total huge pages allocation for all nodes.

Consider mentioning impact of turning off huge pages, or linking to docs on it; otherwise not clear why huge pages shouldn't just be unconditionally off.

Reply:
Same here. It isn't easy to give a complete guide here, but I hope the new version is more explicit.